### DISCUSSION TITLE
can it connect to neovim?

### DISCUSSION THREAD

**Discussion by cStralpt** [Jelmerro/Vieb#540] [Q&A] [answered]
connect to neovim and based on currentt neovim config, i can control/interact with vieb using my own keybindings/use text object to select/copy ttext etc.

**Comment by Jelmerro** [comment, marked as answer]
Yes, you can change the vim editor with the `vimcommand` setting, see `:h vimcommand`. It can be opened from most places (input fields, the navbar etc.) with `<C-i>` (so that's Control + i), see `:h editWithVim` for the action details.
